bigbluebutton-html5/imports/ui/components/whiteboard/component.jsx

```diff
@@ -211,8 +213,19 @@ export default function Whiteboard(props) {
     const invalidShapeType = Object.keys(changedShapes)
       .find(id => !isValidShapeType(changedShapes[id]));
 
-    if (invalidChange || invalidShapeType) {
-      notifyNotAllowedChange(intl);
+    const shapeNumberExceeded = Object.keys(shapes).length > maxNumberOfAnnotations;

```

Comment:
Like this we are allowing the user to select/copy 99 shapes and paste and it will allow, need to do the check the shapes from tldraw document